websites/Z/Zee5/presence.ts

```diff
@@ -0,0 +1,94 @@
+const presence = new Presence({
+		clientId: "1103408220921397349",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+enum Assets {
+	Logo = "https://i.imgur.com/rMcEfBA.jpg",
+	Pause = "https://i.imgur.com/0A75vqT.png",
+	Play = "https://i.imgur.com/Dj5dekr.png",
+}
+
+function getTimestamps(videoTime: number, videoDuration: number): number[] {
+	const startTime = Date.now();
+	return [
+		Math.floor(startTime / 1000),
+		Math.floor(startTime / 1000) - videoTime + videoDuration,
+	];
+}
```

Comment:
why? we have [presence.getTimestamps](https://docs.premid.app/en/dev/presence/class#gettimestampsnumber-number)
https://github.com/PreMiD/Presences/blob/05fa61c3473ee6ae34fe5b5b150d685dfebc76c9/%40types/premid/index.d.ts#L366